MNE/disp3D/3DObjects/brain/brainhemispheretreeitem.cpp

```diff
@@ -105,9 +105,6 @@ BrainHemisphereTreeItem::BrainHemisphereTreeItem(int iType, const QString& text)
 
 BrainHemisphereTreeItem::~BrainHemisphereTreeItem()
 {
-//    delete m_pSurfaceItem;
-//    delete m_pAnnotItem;
-//    delete m_pBrainRTSourceLocDataTreeItem;
```

Comment:
? No deletion required - if those are QObjects, consider using QPointer instead of raw pointers